modules/cnv-add-boot-order-web.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assembly:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+//
+
+[id="cnv-add-boot-order-web_{context}"]
+= Adding items to a boot order list in the web console
+
+Using the web console, add items to a boot order list.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a virtual machine to open the *Virtual Machine Overview* screen.
```

Comment:
This applies to all of the procedures: add a line break in between each step.

Reply:
@ousleyp - When I look into the other cnv files that have procedures, I am not seeing a line break between each step in procedures. One file I viewed was cnv-editing-vm-web.adoc. In the steps for the procedure, each step is immediately followed by the next procedural step without a line break... Could you please clarify, thanks!

Reply:
Clarified via slack exchange. Adding the line breaks.